tools/functional-tester/etcd-tester/lease_stresser.go

```diff
@@ -402,11 +369,12 @@ func (ls *leaseStresser) Cancel() {
 	ls.cancel()
 	ls.runWg.Wait()
 	ls.aliveWg.Wait()
+	ls.conn.Close()
```

Comment:
should we close conn at the cancelled function? in that way, i probably should remove the close at line 113